gridded/pyugrid/ugrid.py

```diff
@@ -237,6 +237,8 @@ def num_vertices(self):
     def nodes(self):
         return self._nodes
 
+    node_coordinates = nodes
```

Comment:
I'll look at the rest of the code, but I'd rather not add aliases for existing names -- and we really don't want to change the names.

Reply:
@ChrisBarker-NOAA see my explanation here: https://github.com/NOAA-ORR-ERD/gridded/issues/65#issuecomment-811621865